packages/components/src/utils/math.js

```diff
@@ -61,6 +56,20 @@ function getPrecision( value ) {
 	return split[ 1 ] !== undefined ? split[ 1 ].length : 0;
 }
 
+/**
+ * Clamps a value based on a min/max range.
+ *
+ * @param {number} value The value.
+ * @param {number} min   The minimum range.
+ * @param {number} max   The maximum range.
+ *
+ * @return {number} The clamped value.
+ */
+export function clamp( value = 0, min = -Infinity, max = Infinity ) {
```

Comment:
Looking at [types for `lodash`'s `clamp`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/lodash/common/number.d.ts#L3-L39), the `min` argument can be omitted (the function seems to have 2 different signatures, one with 3 arguments and one with 2 arguments).

Although it looks like throughout the repo we always use the version with 3 arguments, and so it's probably ok go forward with these changes

Reply:
I actually intentionally didn't replicate this behavior, because I find it ambiguous. Actually, usages like this are one of the primary reasons for my desire to remove any Lodash 😉 It tolerates so much, but that makes the code so ambiguous and error-prone. And we don't use that here, so what better reason to keep it simple.